helm/examples/efs/hpcc-efs/templates/NOTES.txt

```diff
@@ -0,0 +1,22 @@
+{{ $dllPvcName := printf "dll-%s-pvc" (include "hpcc-efs.fullname" $) }}
+{{ $daliPvcName := printf "dali-%s-pvc" (include "hpcc-efs.fullname" $) }}
+{{ $dataPvcName := printf "data-%s-pvc" (include "hpcc-efs.fullname" $) }}
+{{ $sashaPvcName := printf "sasha-%s-pvc" (include "hpcc-efs.fullname" $) }}
+
+# Persistent Volume Claims created: {{ $dllPvcName }}, {{ $daliPvcName }}, {{ $dataPvcName }}, {{ $sashaPvcName }}
+#
+# Pass a values yaml to hpcc helm charts to use these existing claims.
+# e.g.:
+
+storage:
+  dllStorage:
+    existingClaim: "{{ $dllPvcName }}"
+
+  daliStorage:
+    existingClaim: "{{ $daliPvcName }}"
+
+  dataStorage:
+    existingClaim: "{{ $dataPvcName }}"
+
+  sashaStorage:
+    existingClaim: "{{ $sashaPvcName }}"
```

Comment:
Similar issues to : https://github.com/hpcc-systems/HPCC-Platform/pull/14797
Will hold off from reviewing further until gh-14797 is resovled/merged.